cli/run_parser.py

```diff
@@ -12,27 +12,26 @@
 
 sys.path.append("..")
 
-from src.base import (
+from src.base import (  # noqa: E402
```

Comment:
we really do need a quick restructure of these repos so that messing with `sys.path` isn't required any more!